src/test/java/org/springframework/integration/kafka/listener/TestSingleBrokerExternal.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.springframework.integration.kafka.listener;
+
+import java.util.UUID;
+
+import org.junit.After;
+import org.junit.ClassRule;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import org.springframework.integration.kafka.rule.KafkaRule;
+import org.springframework.integration.kafka.rule.KafkaRunning;
+
+/**
+ * @author Marius Bogoevici
+ */
+@Ignore
```

Comment:
Well. The good compromise to `@Ignore` them all, since they have a hard work with filesystem and we don't want to impact CI server so much. Fortunately we already know that they work as expected.